perfkitbenchmarker/publisher.py

```diff
@@ -76,12 +76,11 @@
     None,
     'GCS bucket to upload records to. Bucket must exist.')
 
-flags.DEFINE_list(
+flags.DEFINE_multistring(
     'metadata',
     [],
-    'A list of key-value pairs that will be added to the labels field of all '
-    'samples as metadata. Each key-value pair in the list should be colon '
-    'separated.')
+    'A colon separated key-value pair that will be added to the labels field '
+    'of all samples as metadata. This option can be repeated multiple times.')
```

Comment:
Could you note that kv-pairs can be comma-separated within a `--metadata` specification?